Checks/PWR075/benchmark/solution.f90

```diff
@@ -0,0 +1,58 @@
+! PWR075: Avoid using GNU Fortran extensions
+!
+! The Fortran standard specifies the availability of:
+!
+!   - `cosd` and `sind`, which operate on degrees and real(4) types (Fortran 23).
+!   - `dcos` and `dsin`, which operate on radians and real(8) types (Fortran 77).
+!
+! To replace `dcosd` and `dsind`, let's convert from degrees to radians
+! and then call `dcos` and `dsin`. This ensures that all computations are still
+! performed in real(8).
+
+module trigonometric_utils
+  use iso_c_binding, only : c_double
+  implicit none
+
+  real(kind=c_double), parameter :: pi = 3.141592653589793_c_double
+
+contains
+
+  real(kind=c_double) function dcosd(degrees)
+    real(kind=c_double), intent(in) :: degrees
+    ! `dcos` expects the angle to be in radians, not degrees
+    dcosd = dcos(degrees * pi / 180.0)
```

Comment:
https://tauday.com/tau-manifesto
Couldn't resist.
TLDR: $\tau=circumference/radius=2\pi\approx6.28$ is a constant that makes more sense IMO.
$degrees / 360 = turns$
$turns * \tau = radians$

Feel free to ignore if confusing to those unfamiliar.

Reply:
Wow, it's been years since I last heard from tau :grin: 

> Feel free to ignore if confusing to those unfamiliar.

Indeed, I'd prefer to keep pi for familiarity to most readers.